doc/matter/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _matter_index:
+
+Matter documentation
+####################
+
+This documentation is based on the documentation in the `Matter <https://github.com/nrfconnect/sdk-connectedhomeip>`_ repository fork, and focuses on the nRF Connect platform in Matter.
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+   :caption: Contents:
+
+   nrfconnect_*
+   openthread_*
+   python_*
+   android_*
+   BUILDING
```

Comment:
Could we re-arrange the ordering a little bit? @greg-fer what do you think?

This is my proposition (from the most important from the user point of view to the least).

```suggestion
   nrfconnect_*
   python_chip_controller_building
   python_chip_controller_advanced_usage
   android_*
   openthread_*
   BUILDING
```

Reply:
Here, it would be great if nrfconnect_platform_overview.md could come top of the list, with BUILDING before other nrfconnect_ pages, in my opinion.

Reply:
done